test/e2e/request-queue-with-concurrency/test.mjs

```diff
@@ -0,0 +1,45 @@
+import { initialize, getActorTestDir, pushActor, startActorOnPlatform, expect } from '../tools.mjs';
+import { Actor } from 'apify';
+import { log } from 'crawlee';
+import { setTimeout } from 'node:timers/promises';
+
+if (process.env.STORAGE_IMPLEMENTATION === 'PLATFORM') {
```

Comment:
can we keep it enabled for memory/local storage too?

Reply:
I'm afraid that the test code is dependent on the Actor to run on the platform. Besides, I'm not sure if parallel access to memory storage is something we need to test.

Reply:
We'd have to run the Actor in subprocesses to achieve parallel execution, which seems like non-trivial work to test a super niche use case. For the "normal" use case, the `request-queue-zero-concurrency` test should suffice.